2020/13xxx/CVE-2020-13671.json

```diff
@@ -1,18 +1,87 @@
 {
```

Comment:
{} is not allowed for {'reference_data': [{'refsource': 'CONFIRM', 'url': 'https://www.drupal.org/sa-core-2020-012', 'name': ''}]}